### ISSUE TITLE
Document the project in English

### ISSUE THREAD

**Issue by axe312ger** [dsstio/ecw2tiles#1]
You realized by any fact this is an english platform?

Sorry, but creating language barriers is not the idea of open source.

**Comment by pbock** [comment]
A valid point, thanks!

I gather that you speak both German and English. If the project is useful to you and you have a few minutes to spare to make it accessible to a wider audience, perhaps you'd like to help out with an English translation. We welcome pull requests.

**Comment by axe312ger** [comment]
Thanks for the rapid feedback. Very nice, but have to admit I didn't test it yet, even when it looks pretty interesting.

Just wanted to point out that this might be useful for other non-german speakers from all over the planet :)

**Comment by pbock** [comment]
Yup, and documentation is definitely something we can improve (especially in a project like this one that's more tutorial than source code) – it's just not at the very top of our list of priorities at the moment, I'm afraid.

**Comment by axe312ger** [comment]
just a small hint: consider adding eslint + non-tab code style to your list of priorities 😎
